docs/pages/deployment/key-rotation.rst

```diff
@@ -3,6 +3,10 @@
 Key rotation procedure
 ######################
 
+.. warning::
+    Rotating keys currently prevents private transactions (e.g. ``NutsAuthorizationCredential``) from being received properly:
```

Comment:
It's not that black and white. Only happens if keys rotate and credentials are in transit at the same time.

Reply:
Which you don't know, so it can't be used safely in any circumstance?